packages/abstract-signer/src.ts/index.ts

```diff
@@ -132,16 +132,6 @@ export abstract class Signer {
         return network.chainId;
     }
 
-    async getGasPrice(): Promise<BigNumber> {
-        this._checkProvider("getGasPrice");
-        return await this.provider.getGasPrice();
-    }
```

Comment:
Lets leave the `getGasPrice` method here, since we have it on the `provider` level. It just throws `not implemented` right now